app/views/track.js

```diff
@@ -20,6 +20,7 @@ const TrackView = Mn.LayoutView.extend({
     },
 
     events: {
+        'mousedown .drag': 'dragstart',
```

Comment:
With a global listener, this event should be in `tracks` composite view, so we've got only one listener on top for mousedown, not one for each row in the list.

As we need to identify which row is dragged, we could add a data-attr (like `data-id`) on the `this.el` that contains the model id. This way, on https://github.com/cozy-labs/cozy-music/pull/82/files#diff-6a3ff9bec0a7354a982b03fc908248eaR44, we just have to extract the model id from `event.target.dataset.id` and get the model from it.